COLLABORATOR_GUIDE.md

```diff
@@ -80,12 +102,33 @@ elevate discussion to the TSC for resolution (see below).
 
 Breaking changes (that is, pull requests that require an increase in
 the major version number, known as `semver-major` changes) must be
-elevated for review by the TSC. This does not necessarily mean that the
-PR must be put onto the TSC meeting agenda. If multiple TSC members
-approve (`LGTM`) the PR and no Collaborators oppose the PR, it can be
-landed. Where there is disagreement among TSC members or objections
-from one or more Collaborators, `semver-major` pull requests should be
-put on the TSC meeting agenda.
+[elevated for review by the TSC](#involving-the-tsc).
+This does not necessarily mean that the PR must be put onto the TSC meeting
+agenda. If multiple TSC members approve (`LGTM`) the PR and no Collaborators
+oppose the PR, it can be landed. Where there is disagreement among TSC members
+or objections from one or more Collaborators, `semver-major` pull requests
+should be put on the TSC meeting agenda.
+
+### Waiting for Approvals
+
+Before landing pull requests, sufficient time should be left for input
+from other Collaborators. In general, leave at least 48 hours during the
+week and 72 hours over weekends to account for international time
+differences and work schedules. However, certain types of pull requests
+can be fast-tracked and may be landed after a shorter delay:
+
+* Trivial changes, for example, those fixing minor bugs or improving
+  performance without affecting API or causing other wide-reaching impact.
+* Focused changes that affect only documentation and/or the test suite.
+  `code-and-learn` and `good-first-issue` pull requests typically fall
+  into this category.
+* Changes that revert commit(s) and/or fix regressions.
+
+When a pull request is deemed suitable to be fast-tracked, label it with
+`fast-track`. The pull request can be landed once more than 3 collaborators
```

Comment:
The number 3 here needs more discussion

Reply:
I'm 👍 on fast-tracking everything that fixed a regression

Reply:
I would go with 2, and be more explicit:
`The PR can be landed after 2 or more collaborators (excluding the author) have approved it.`